test/storage-luatest/storage_1_1_1_test.lua

```diff
@@ -195,3 +195,28 @@ rebalancer_recovery_group.test_no_logs_while_unsuccess_recovery = function(g)
     wait_for_bucket_is_transferred(g.replica_2_a, g.replica_1_a,
                                    hanged_bucket_id_2)
 end
+
+rebalancer_recovery_group.test_rebalancer_routes_logging = function(g)
+    local moved_bucket_from_2 = vtest.storage_first_bucket(g.replica_2_a)
+    start_bucket_move(g.replica_2_a, g.replica_1_a, moved_bucket_from_2)
+    local moved_bucket_from_3 = vtest.storage_first_bucket(g.replica_3_a)
+    start_bucket_move(g.replica_3_a, g.replica_1_a, moved_bucket_from_3)
+    t.helpers.retrying({timeout = 60}, function()
+        g.replica_1_a:exec(function()
+            ivshard.storage.rebalancer_wakeup()
+        end)
+        t.assert(g.replica_1_a:grep_log(
+            'Apply rebalancer routes with 1 workers'))
+        end)
+    local rebalancer_routes_msg = string.format("{\"%s\":{\"%s\":1,\"%s\":1}}",
+                                             g.replica_1_a:replicaset_uuid(),
+                                             g.replica_3_a:replicaset_uuid(),
+                                             g.replica_2_a:replicaset_uuid())
+    t.assert(g.replica_1_a:grep_log(rebalancer_routes_msg))
+    start_bucket_move(g.replica_1_a, g.replica_2_a, moved_bucket_from_2)
+    start_bucket_move(g.replica_1_a, g.replica_3_a, moved_bucket_from_3)
```

Comment:
You're moving the buckets with rebalancer, why do you need to manually move them then?